Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11-jdk
+WORKDIR /app
+COPY . .
+RUN chmod +x gradlew
+CMD sh gradlew run
```

Comment:
I tried to run the docker compose version locally and it failed.  So please check when you get some time 
```
playground-for-kotlin-playground-for-kotlin-1  | Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
playground-for-kotlin-playground-for-kotlin-1  | > Task :compileKotlin
playground-for-kotlin-playground-for-kotlin-1  | 
playground-for-kotlin-playground-for-kotlin-1  | The message received from the daemon indicates that the daemon has disappeared.
playground-for-kotlin-playground-for-kotlin-1  | Build request sent: Build{id=5d6608e2-1ee2-4a9d-9a8c-f1cd1f909b84, currentDir=/app}
playground-for-kotlin-playground-for-kotlin-1  | Attempting to read last messages from the daemon log...
playground-for-kotlin-playground-for-kotlin-1  | Daemon pid: 37
playground-for-kotlin-playground-for-kotlin-1  |   log file: /root/.gradle/daemon/6.8/daemon-37.out.log
playground-for-kotlin-playground-for-kotlin-1  | ----- Last  20 lines from daemon log file - daemon-37.out.log -----
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.838+0000 [ERROR] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] Command execution: started DaemonCommandExecution[command = Build{id=5d6608e2-1ee2-4a9d-9a8c-f1cd1f909b84, currentDir=/app}, connection = DefaultDaemonConnection: socket connection from /127.0.0.1:33347 to /127.0.0.1:58438] after 0.0 minutes of idle
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.838+0000 [INFO] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Marking the daemon as busy, address: [d2907264-2371-4060-a351-67a47d13da1e port:33347, addresses:[localhost/127.0.0.1]]
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.838+0000 [DEBUG] [org.gradle.launcher.daemon.registry.PersistentDaemonRegistry] Marking busy by address: [d2907264-2371-4060-a351-67a47d13da1e port:33347, addresses:[localhost/127.0.0.1]]
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.839+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.839+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.839+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.840+0000 [DEBUG] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] resetting idle timer
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.840+0000 [DEBUG] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] daemon is running. Sleeping until state changes.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.841+0000 [INFO] [org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy] Daemon is about to start building Build{id=5d6608e2-1ee2-4a9d-9a8c-f1cd1f909b84, currentDir=/app}. Dispatching build started information...
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.841+0000 [DEBUG] [org.gradle.launcher.daemon.server.SynchronizedDispatchConnection] thread 20: dispatching org.gradle.launcher.daemon.protocol.BuildStarted@45f7f00c
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.842+0000 [DEBUG] [org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment] Configuring env variables: [PATH, HOSTNAME, JAVA_HOME, OLDPWD, PWD, JAVA_VERSION, LANG, HOME]
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.843+0000 [DEBUG] [org.gradle.launcher.daemon.server.SynchronizedDispatchConnection] thread 18: received class org.gradle.launcher.daemon.protocol.CloseInput
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.844+0000 [DEBUG] [org.gradle.launcher.daemon.server.DefaultDaemonConnection] thread 18: Received IO message from client: org.gradle.launcher.daemon.protocol.CloseInput@540c7277
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.846+0000 [DEBUG] [org.gradle.launcher.daemon.server.exec.LogToClient] About to start relaying all logs to the client via the connection.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.846+0000 [INFO] [org.gradle.launcher.daemon.server.exec.LogToClient] The client will now receive all logging from the daemon (pid: 37). The daemon log file: /root/.gradle/daemon/6.8/daemon-37.out.log
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.847+0000 [INFO] [org.gradle.launcher.daemon.server.exec.LogAndCheckHealth] Starting build in new daemon [memory: 512 MiB]
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.848+0000 [INFO] [org.gradle.launcher.daemon.server.exec.ForwardClientInput] Closing daemon's stdin at end of input.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.849+0000 [INFO] [org.gradle.launcher.daemon.server.exec.ForwardClientInput] The daemon will no longer process any standard input.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.851+0000 [DEBUG] [org.gradle.launcher.daemon.server.exec.ExecuteBuild] The daemon has started executing the build.
playground-for-kotlin-playground-for-kotlin-1  | 2022-03-10T10:47:07.851+0000 [DEBUG] [org.gradle.launcher.daemon.server.exec.ExecuteBuild] Executing build with daemon context: DefaultDaemonContext[uid=d02fbe2a-a850-4363-bda9-843b2e6f54d0,javaHome=/usr/local/openjdk-11,daemonRegistryDir=/root/.gradle/daemon,pid=37,idleTimeout=10800000,priority=NORMAL,daemonOpts=--add-opens,java.base/java.util=ALL-UNNAMED,--add-opens,java.base/java.lang=ALL-UNNAMED,--add-opens,java.base/java.lang.invoke=ALL-UNNAMED,--add-opens,java.prefs/java.util.prefs=ALL-UNNAMED,-XX:MaxMetaspaceSize=256m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=UTF-8,-Duser.country,-Duser.language=en,-Duser.variant]
playground-for-kotlin-playground-for-kotlin-1  | ----- End of the daemon log -----
playground-for-kotlin-playground-for-kotlin-1  | 
playground-for-kotlin-playground-for-kotlin-1  | 
playground-for-kotlin-playground-for-kotlin-1  | FAILURE: Build failed with an exception.
playground-for-kotlin-playground-for-kotlin-1  | 
playground-for-kotlin-playground-for-kotlin-1  | * What went wrong:
playground-for-kotlin-playground-for-kotlin-1  | Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)
playground-for-kotlin-playground-for-kotlin-1  | 
playground-for-kotlin-playground-for-kotlin-1  | * Try:
playground-for-kotlin-playground-for-kotlin-1  | Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
playground-for-kotlin-playground-for-kotlin-1  | 
playground-for-kotlin-playground-for-kotlin-1  | * Get more help at https://help.gradle.org/
```

Reply:
@christyjacob4 Please try again now